src/polychord/nested_sampling.F90

```diff
@@ -191,11 +191,7 @@ end subroutine dumper
             end if
 
             ! Intialise the run by setting all of the relevant run time info, and generating live points
-            if (settings%generate_from_seed) then
```

Comment:
since `settings%generate_from_seed` is no longer used, it should be removed from settings too